packages/cdk-routing-split/test/steps/when.ts

```diff
@@ -9,6 +9,7 @@ import { Request as DeleteRequest } from '../../src/delete/request';
 import { Request as UpdateRequest } from '../../src/update/request';
 
 export const we_invoke_add = async (body: AddRequest) => {
+    console.log('add - body', body);
```

Comment:
A new console.log was added. Please make sure you're code reviewing your changes before committing them.

Reply:
Oh, this is a test file, sorry my mistake.